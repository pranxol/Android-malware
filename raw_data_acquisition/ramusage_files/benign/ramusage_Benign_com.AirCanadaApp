Applications Memory Usage (kB):
Uptime: 515303 Realtime: 515292

** MEMINFO in pid 5532 [com.AirCanadaApp] **
                   Pss      Pss   Shared  Private   Shared  Private  Swapped     Heap     Heap     Heap
                 Total    Clean    Dirty    Dirty    Clean    Clean    Dirty     Size    Alloc     Free
                ------   ------   ------   ------   ------   ------   ------   ------   ------   ------
  Native Heap     3887        0     1648     3848        0        0        0    16384     6042    10341
  Dalvik Heap    15978        0    17712    15556        0        0        0    40003    31842     8161
 Dalvik Other      444        0        8      444        0        0        0                           
        Stack      152        0        0      152        0        0        0                           
      Gfx dev    15348        0        0    15348        0        0        0                           
    Other dev        4        0       36        0        0        4        0                           
     .so mmap     3548     2208     1952      676     8608     2208        0                           
    .apk mmap       71        8        0        0     1012        8        0                           
    .ttf mmap      274      200        0        0      332      200        0                           
    .dex mmap      892      888        0        0        8      888        0                           
    .oat mmap     1035      132        0        0    11500      132        0                           
    .art mmap     1324        0     1044     1120     7280        0        0                           
   Other mmap       23        0        8        8      228        0        0                           
   EGL mtrack    13888        0        0    13888        0        0        0                           
    GL mtrack    19372        0        0    19372        0        0        0                           
      Unknown     9112        0       76     9112        0        0        0                           
        TOTAL    85352     3436    22484    79524    28968     3440        0    56387    37884    18502
 
 Dalvik Details
        .Heap     1476        0        0     1476        0        0        0                           
         .LOS    13508        0        0    13508        0        0        0                           
          .GC      356        0        0      356        0        0        0                           
      .Zygote      830        0    17712      408        0        0        0                           
   .NonMoving      164        0        0      164        0        0        0                           
 .IndirectRef       88        0        8       88        0        0        0                           
 
 Objects
               Views:       40         ViewRootImpl:        2
         AppContexts:        3           Activities:        1
              Assets:        2        AssetManagers:        2
       Local Binders:        8        Proxy Binders:       12
       Parcel memory:        3         Parcel count:       12
    Death Recipients:        0      OpenSSL Sockets:        1
 
 SQL
         MEMORY_USED:        0
  PAGECACHE_OVERFLOW:        0          MALLOC_SIZE:        0
 
