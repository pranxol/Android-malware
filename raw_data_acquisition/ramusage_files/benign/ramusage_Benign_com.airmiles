Applications Memory Usage (kB):
Uptime: 645898 Realtime: 645887

** MEMINFO in pid 6446 [com.airmiles] **
                   Pss      Pss   Shared  Private   Shared  Private  Swapped     Heap     Heap     Heap
                 Total    Clean    Dirty    Dirty    Clean    Clean    Dirty     Size    Alloc     Free
                ------   ------   ------   ------   ------   ------   ------   ------   ------   ------
  Native Heap     6578        0     1672     6540        0        0        0    20480     8954    11525
  Dalvik Heap    10590        0    17692    10188        0        0        0    36021    27860     8161
 Dalvik Other      432        0        8      432        0        0        0                           
        Stack      232        0        0      232        0        0        0                           
      Gfx dev      764        0        0      764        0        0        0                           
    Other dev        8        0       40        0        0        8        0                           
     .so mmap     1072       12     1940      316     8640       12        0                           
    .jar mmap        0        0        0        0        4        0        0                           
    .apk mmap      101       28        0        0     1064       28        0                           
    .ttf mmap        3        0        0        0       12        0        0                           
    .dex mmap     6269     6248        0        0       84     6248        0                           
    .oat mmap      953       52        0        0    12028       52        0                           
    .art mmap      994        4     1276      788     7388        4        0                           
   Other mmap      224        0        8        4      368      192        0                           
   EGL mtrack    13888        0        0    13888        0        0        0                           
    GL mtrack     5500        0        0     5500        0        0        0                           
      Unknown      152        0       80      152        0        0        0                           
        TOTAL    47760     6344    22716    38804    29588     6544        0    56501    36814    19686
 
 Dalvik Details
        .Heap     1948        0        0     1948        0        0        0                           
         .LOS     7296        0        0     7296        0        0        0                           
          .GC      384        0        0      384        0        0        0                           
      .Zygote      830        0    17692      428        0        0        0                           
   .NonMoving      516        0        0      516        0        0        0                           
 .IndirectRef       48        0        8       48        0        0        0                           
 
 Objects
               Views:       44         ViewRootImpl:        2
         AppContexts:        4           Activities:        2
              Assets:        2        AssetManagers:        2
       Local Binders:       52        Proxy Binders:       21
       Parcel memory:        7         Parcel count:       28
    Death Recipients:        0      OpenSSL Sockets:        2
 
 SQL
         MEMORY_USED:     1123
  PAGECACHE_OVERFLOW:      672          MALLOC_SIZE:       62
 
 DATABASES
      pgsz     dbsz   Lookaside(b)          cache  Dbname
         4      224             44      79/111/25  /data/data/com.airmiles/databases/airmiles.db
         4       20             38        21/35/8  /data/data/com.airmiles/databases/ua_analytics.db
         4       20             35        84/34/7  /data/data/com.airmiles/databases/ua_preferences.db
         4       28             44        17/41/8  /data/data/com.airmiles/databases/google_analytics_v4.db
         4       24             45         1/31/4  /data/data/com.airmiles/databases/ua_richpush.db
