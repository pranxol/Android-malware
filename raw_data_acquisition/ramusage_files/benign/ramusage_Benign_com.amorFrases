Applications Memory Usage (kB):
Uptime: 562474 Realtime: 562470

** MEMINFO in pid 4752 [com.amorFrases] **
                   Pss      Pss   Shared  Private   Shared  Private  Swapped     Heap     Heap     Heap
                 Total    Clean    Dirty    Dirty    Clean    Clean    Dirty     Size    Alloc     Free
                ------   ------   ------   ------   ------   ------   ------   ------   ------   ------
  Native Heap     4257        0     1544     4224        0        0        0     5904     5372      531
  Dalvik Heap     4094        0     8828     3892        0        0        0    14611    10989     3622
 Dalvik Other      340        0        8      340        0        0        0                           
        Stack      376        0        0      376        0        0        0                           
       Ashmem      140        0       20      128        4        0        0                           
      Gfx dev      112        0        0      112        0        0        0                           
    Other dev        4        0       32        0        0        4        0                           
     .so mmap    19666    16620     1920      812    10628    16620        0                           
    .apk mmap     1710     1476        0        0     1264     1476        0                           
    .ttf mmap      180        0        0        0      512        0        0                           
    .dex mmap     2408     1948        0        0      924     1948        0                           
    .oat mmap      910      168        0        0    11088      168        0                           
    .art mmap     1268        0     1020     1068     7560        0        0                           
   Other mmap      830        0        8        8      192      780        0                           
    GL mtrack      668        0        0      668        0        0        0                           
      Unknown    13516        0       72    13516        0        0        0                           
        TOTAL    50479    20212    13452    25144    32172    20996        0    20515    16361     4153
 
 Dalvik Details
        .Heap     1460        0        0     1460        0        0        0                           
         .LOS     1716        0        0     1716        0        0        0                           
          .GC      252        0        0      252        0        0        0                           
      .Zygote      398        0     8828      196        0        0        0                           
   .NonMoving      520        0        0      520        0        0        0                           
 .IndirectRef       88        0        8       88        0        0        0                           
 
 Objects
               Views:       18         ViewRootImpl:        1
         AppContexts:        3           Activities:        1
              Assets:        3        AssetManagers:        3
       Local Binders:       19        Proxy Binders:       21
       Parcel memory:        5         Parcel count:       21
    Death Recipients:        0      OpenSSL Sockets:        0
 
 SQL
         MEMORY_USED:        0
  PAGECACHE_OVERFLOW:        0          MALLOC_SIZE:        0
 
