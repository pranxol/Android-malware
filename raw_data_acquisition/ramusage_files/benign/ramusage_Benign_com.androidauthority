Applications Memory Usage (kB):
Uptime: 518341 Realtime: 518328

** MEMINFO in pid 5766 [com.androidauthority] **
                   Pss      Pss   Shared  Private   Shared  Private  Swapped     Heap     Heap     Heap
                 Total    Clean    Dirty    Dirty    Clean    Clean    Dirty     Size    Alloc     Free
                ------   ------   ------   ------   ------   ------   ------   ------   ------   ------
  Native Heap     6642        0     1652     6604        0        0        0    16384     7872     8511
  Dalvik Heap     6135        0    17668     5724        0        0        0    28799    21630     7169
 Dalvik Other      500        0        8      500        0        0        0                           
        Stack      364        0        0      364        0        0        0                           
       Ashmem      128        0        0      128        0        0        0                           
      Gfx dev     4252        0        0     4252        0        0        0                           
    Other dev        4        0       36        0        0        4        0                           
     .so mmap     6748     3296     1920      784    14644     3296        0                           
    .apk mmap      524      112        0        0     2012      112        0                           
    .ttf mmap      503      472        0        0      144      472        0                           
    .dex mmap     4678     4348        0        0      868     4348        0                           
    .oat mmap     1660      392        0        0    13708      392        0                           
    .art mmap     1493        0     1016     1296     7328        0        0                           
   Other mmap      213        0        8        8      640       68        0                           
   EGL mtrack    13888        0        0    13888        0        0        0                           
    GL mtrack     7856        0        0     7856        0        0        0                           
      Unknown     9140        0       64     9140        0        0        0                           
        TOTAL    64728     8620    22372    50544    39344     8692        0    45183    29502    15680
 
 Dalvik Details
        .Heap     2692        0        0     2692        0        0        0                           
         .LOS     1784        0        0     1784        0        0        0                           
          .GC      392        0        0      392        0        0        0                           
      .Zygote      863        0    17668      452        0        0        0                           
   .NonMoving      796        0        0      796        0        0        0                           
 .IndirectRef      108        0        8      108        0        0        0                           
 
 Objects
               Views:      193         ViewRootImpl:        2
         AppContexts:        3           Activities:        1
              Assets:        3        AssetManagers:        3
       Local Binders:       15        Proxy Binders:       20
       Parcel memory:        7         Parcel count:       28
    Death Recipients:        3      OpenSSL Sockets:        0
 
 SQL
         MEMORY_USED:      476
  PAGECACHE_OVERFLOW:      328          MALLOC_SIZE:       90
 
 DATABASES
      pgsz     dbsz   Lookaside(b)          cache  Dbname
         4      160            394     709/119/25  /data/data/com.androidauthority/databases/data2.db
