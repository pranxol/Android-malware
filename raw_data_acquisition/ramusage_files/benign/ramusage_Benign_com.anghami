Applications Memory Usage (kB):
Uptime: 675161 Realtime: 675148

** MEMINFO in pid 6900 [com.anghami] **
                   Pss      Pss   Shared  Private   Shared  Private  Swapped     Heap     Heap     Heap
                 Total    Clean    Dirty    Dirty    Clean    Clean    Dirty     Size    Alloc     Free
                ------   ------   ------   ------   ------   ------   ------   ------   ------   ------
  Native Heap    14210        0     1628    14176        0        0        0    32768    15516    17251
  Dalvik Heap    30259        0    17620    29884        0        0        0    53854    45693     8161
 Dalvik Other      772        0        8      772        0        0        0                           
        Stack      560        0        0      560        0        0        0                           
      Gfx dev     7604        0        0     7604        0        0        0                           
    Other dev       20        0       40        0        0       20        0                           
     .so mmap     3849     2292     1928      700     9932     2292        0                           
    .apk mmap      451       92        0        0     1664       92        0                           
    .ttf mmap      189        0        0        0      652        0        0                           
    .dex mmap    14464    14460        0        0        8    14460        0                           
    .oat mmap     2211      728        0        0    16064      728        0                           
    .art mmap     1842       12      860     1660     7048       12        0                           
   Other mmap     2240        0        8        8      540     2176        0                           
   EGL mtrack    13888        0        0    13888        0        0        0                           
    GL mtrack    11340        0        0    11340        0        0        0                           
      Unknown    11540        0       76    11540        0        0        0                           
        TOTAL   115439    17584    22168    92132    35908    19780        0    86622    61209    25412
 
 Dalvik Details
        .Heap    11760        0        0    11760        0        0        0                           
         .LOS    15832        0        0    15832        0        0        0                           
          .GC      584        0        0      584        0        0        0                           
      .Zygote      875        0    17620      500        0        0        0                           
   .NonMoving     1792        0        0     1792        0        0        0                           
 .IndirectRef      188        0        8      188        0        0        0                           
 
 Objects
               Views:      387         ViewRootImpl:        1
         AppContexts:        6           Activities:        2
              Assets:        2        AssetManagers:        2
       Local Binders:       62        Proxy Binders:       30
       Parcel memory:       17         Parcel count:       70
    Death Recipients:        0      OpenSSL Sockets:        8
 
 SQL
         MEMORY_USED:      735
  PAGECACHE_OVERFLOW:      368          MALLOC_SIZE:       62
 
 DATABASES
      pgsz     dbsz   Lookaside(b)          cache  Dbname
         4      104            283      208/82/15  /data/data/com.anghami/databases/anghami.db
         4       20             40        20/36/9  /data/data/com.anghami/no_backup/com.urbanairship.databases/ua_analytics.db
         4       24             99         3/33/6  /data/data/com.anghami/no_backup/com.urbanairship.databases/ua_richpush.db
         4       20             35       160/34/7  /data/data/com.anghami/no_backup/com.urbanairship.databases/ua_preferences.db
