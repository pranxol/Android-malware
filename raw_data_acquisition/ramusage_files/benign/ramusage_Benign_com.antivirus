Applications Memory Usage (kB):
Uptime: 543525 Realtime: 543512

** MEMINFO in pid 6291 [com.antivirus] **
                   Pss      Pss   Shared  Private   Shared  Private  Swapped     Heap     Heap     Heap
                 Total    Clean    Dirty    Dirty    Clean    Clean    Dirty     Size    Alloc     Free
                ------   ------   ------   ------   ------   ------   ------   ------   ------   ------
  Native Heap     4804        0     1656     4768        0        0        0    12288     6859     5428
  Dalvik Heap     4412        0    17680     4028        0        0        0    26587    19972     6615
 Dalvik Other      520        0        8      520        0        0        0                           
        Stack      172        0        0      172        0        0        0                           
      Gfx dev     1268        0        0     1268        0        0        0                           
    Other dev        4        0       40        0        0        4        0                           
     .so mmap     3964     2524     1944      780     8796     2524        0                           
    .apk mmap      456      168        0        0     1512      168        0                           
    .ttf mmap      128        0        0        0      572        0        0                           
    .dex mmap     5180     5176        0        0        8     5176        0                           
    .oat mmap      897      140        0        0    11928      140        0                           
    .art mmap     1353        0     1032     1168     7332        0        0                           
   Other mmap       29        0        8        8      316        0        0                           
   EGL mtrack    13888        0        0    13888        0        0        0                           
    GL mtrack     5644        0        0     5644        0        0        0                           
      Unknown    10036        0       76    10036        0        0        0                           
        TOTAL    52755     8008    22444    42280    30464     8012        0    38875    26831    12043
 
 Dalvik Details
        .Heap     2880        0        0     2880        0        0        0                           
         .LOS       40        0        0       40        0        0        0                           
          .GC      400        0        0      400        0        0        0                           
      .Zygote      824        0    17680      440        0        0        0                           
   .NonMoving      668        0        0      668        0        0        0                           
 .IndirectRef      120        0        8      120        0        0        0                           
 
 Objects
               Views:       22         ViewRootImpl:        1
         AppContexts:        4           Activities:        1
              Assets:        2        AssetManagers:        2
       Local Binders:       32        Proxy Binders:       23
       Parcel memory:        5         Parcel count:       20
    Death Recipients:        0      OpenSSL Sockets:        0
 
 SQL
         MEMORY_USED:      135
  PAGECACHE_OVERFLOW:       56          MALLOC_SIZE:       62
 
 DATABASES
      pgsz     dbsz   Lookaside(b)          cache  Dbname
         4       24             25        30/34/5  /data/data/com.antivirus/databases/apps
