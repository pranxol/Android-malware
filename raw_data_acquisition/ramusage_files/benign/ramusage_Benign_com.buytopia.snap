Applications Memory Usage (kB):
Uptime: 512250 Realtime: 512238

** MEMINFO in pid 5810 [com.buytopia.snap] **
                   Pss      Pss   Shared  Private   Shared  Private  Swapped     Heap     Heap     Heap
                 Total    Clean    Dirty    Dirty    Clean    Clean    Dirty     Size    Alloc     Free
                ------   ------   ------   ------   ------   ------   ------   ------   ------   ------
  Native Heap     8099        0     1676     8060        0        0        0    16384    10630     5753
  Dalvik Heap    16551        0    17700    16140        0        0        0    40254    32093     8161
 Dalvik Other      524        0        8      524        0        0        0                           
        Stack      264        0        0      264        0        0        0                           
      Gfx dev     2156        0        0     2156        0        0        0                           
    Other dev        4        0       40        0        0        4        0                           
     .so mmap     3723     2228     1940      688     9164     2228        0                           
    .apk mmap      695      416        0        0     1632      416        0                           
    .ttf mmap      312      244        0        0      264      244        0                           
    .dex mmap     6496     6492        0        0        8     6492        0                           
    .oat mmap     1246      156        0        0    13408      156        0                           
    .art mmap     1460        0      992     1264     7256        0        0                           
   Other mmap       25        0        8        8      252        0        0                           
   EGL mtrack    13888        0        0    13888        0        0        0                           
    GL mtrack     6420        0        0     6420        0        0        0                           
      Unknown     9312        0       76     9312        0        0        0                           
        TOTAL    71175     9536    22440    58724    31984     9540        0    56638    42723    13914
 
 Dalvik Details
        .Heap     2644        0        0     2644        0        0        0                           
         .LOS    12332        0        0    12332        0        0        0                           
          .GC      392        0        0      392        0        0        0                           
      .Zygote      831        0    17700      420        0        0        0                           
   .NonMoving      744        0        0      744        0        0        0                           
 .IndirectRef      132        0        8      132        0        0        0                           
 
 Objects
               Views:       42         ViewRootImpl:        1
         AppContexts:        3           Activities:        1
              Assets:        8        AssetManagers:        8
       Local Binders:       38        Proxy Binders:       20
       Parcel memory:        7         Parcel count:       30
    Death Recipients:        1      OpenSSL Sockets:        2
 
 SQL
         MEMORY_USED:      426
  PAGECACHE_OVERFLOW:      176          MALLOC_SIZE:       62
 
 DATABASES
      pgsz     dbsz   Lookaside(b)          cache  Dbname
         4       36             99        11/41/6  /data/data/com.buytopia.snap/databases/SnapDB.sqlite
         4       20             27        74/32/5  /data/data/com.buytopia.snap/databases/ua_preferences.db
         4       20             35        39/35/8  /data/data/com.buytopia.snap/databases/ua_analytics.db
 
 Asset Allocations
    zip:/data/app/com.buytopia.snap-1/base.apk:/assets/OpenSans-Bold.ttf: 219K
    zip:/data/app/com.buytopia.snap-1/base.apk:/assets/OpenSans-Regular.ttf: 212K
    zip:/data/app/com.buytopia.snap-1/base.apk:/assets/OpenSans-Semibold.ttf: 216K
    zip:/data/app/com.buytopia.snap-1/base.apk:/assets/OpenSans-Semibold.ttf: 216K
    zip:/data/app/com.buytopia.snap-1/base.apk:/assets/OpenSans-Light.ttf: 217K
    zip:/data/app/com.buytopia.snap-1/base.apk:/assets/OpenSans-Semibold.ttf: 216K
